modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/osb-site-initializer-customer-portal-test/src/testFunctional/macros/CP2ActivationKeys.macro

```diff
@@ -1,5 +1,23 @@
 definition {
 
+	macro assertDropDownItemIsDesabled {
+		var key_dropDownItem = "${dropdownItem}";
+
+		AssertElementPresent(locator1 = "CP2ActivationKeys#DESABLED_DROPDOWN_ITEM_BUTTON");
+	}
+
+	macro assertDropDownItemIsEnabled {
```

Comment:
Mathaus, please change all occurrences of "DropDown" to "Dropdown" or "dropdown" in order to keep consistency.